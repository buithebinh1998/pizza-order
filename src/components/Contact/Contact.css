.contact-article {
  display: flex;
  justify-content: center;
}

h1 {
  color: #033a56;
  text-align: center;
}

h2 {
  color: #033a56;
  text-align: center;
}

.contact-article img {
  margin-top: 1rem;
}

.contact-article a {
  color: #033a56;
  font-weight: bold;
}

.contact-article a:visited {
  text-decoration: none;
  color: #033a56;
}

.contact-article section {
  width: 40rem;
  display: flex;
  flex-direction: column;
  font-size: 1.2rem;
  background-color: white;
  border: none;
  align-items: center;
  margin: 1rem;
}

.contact-article section div {
  margin: 1.8rem 0;
}

.contact-article section img {
  max-width: 100%;
  height: auto;
}

@media (max-width: 992px) {
  .contact-article section {
    font-size: 1rem;
    margin: 0.8rem;
  }

  .contact-article section div {
    margin: 1.2rem 0;
  }

  .contact-article section img {
    width: 50%;
  }
}

@media (max-width: 768px) {
  h1 {
    font-size: 1.5rem;
  }

  h2 {
    font-size: 1.2rem;
  }

  .contact-article section {
    font-size: 0.9rem;
    margin: 0.7rem;
    width: 90%;
  }

  .contact-article section div {
    margin: 1rem 0;
  }

  .contact-article section img {
    width: 45%;
  }
}

@media (max-width: 576px) {
  h1 {
    font-size: 1rem;
  }

  h2 {
    font-size: 0.8rem;
  }

  .contact-article section {
    font-size: 0.8rem;
    margin: 0.6rem;
  }

  .contact-article section div {
    margin: 0.8rem 0;
  }

  .contact-article section img {
    width: 30%;
  }
}

@media (max-width: 320px) {
  h1 {
    font-size: 0.8rem;
  }

  h2 {
    font-size: 0.6rem;
  }

  .contact-article section {
    font-size: 0.6rem;
    margin: 0.4rem;
    width: 95%;
  }

  .contact-article section div {
    margin: 0.6rem 0;
  }
}
