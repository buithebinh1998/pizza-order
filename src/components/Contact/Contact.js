import React from "react";
import "./Contact.css";
import telIcon from "./../../assets/img/tel-icon.png";
import fbIcon from "./../../assets/img/facebook.png";
const Contact = () => {
  return (
    <>
      <h1>CONTACT</h1>
      <h2>
        Want to get in touch? We'd love to hear from you. Here's how you can
        contact us.
      </h2>
      <article className="contact-article">
        <section>
          <img src={telIcon} alt="" />
          <div style={{ fontWeight: "bold" }}>Talk to us</div>
          <div>
            Do you have any questions, or you want to call delivery by phone?
            <br />
            Just pick up the phone to call us for more information.
          </div>
          <div>
            Hotline: <a href="tel:19002707">1900-2707</a>
          </div>
        </section>

        <section>
          <img src={fbIcon} alt="" />
          <div style={{ fontWeight: "bold" }}>Chat with us</div>
          <div>
            Do you have any questions, or you want to notify about new
            promotion?
            <br />
            Just like our Facebook fanpage or chat with us on Facebook.
          </div>
          <div>
            Join our fanpage{" "}
            <a
              href="https://www.facebook.com/pycozza"
              target="_blank"
              rel="noopener noreferrer"
            >
              here
            </a>
            .
          </div>
        </section>
      </article>
    </>
  );
};

export default Contact;
