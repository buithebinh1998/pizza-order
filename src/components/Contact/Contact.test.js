import { testSnapshots } from "../../utils/test.util";

import Contact from "./Contact";

describe("<Contact / >", () => {
  testSnapshots(Contact, [
    {
      props: {},
      description: "default render"
    }
  ]);
});