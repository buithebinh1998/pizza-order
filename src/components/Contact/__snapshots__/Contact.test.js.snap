// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact / > snapshots default render 1`] = `
Array [
  <h1>
    CONTACT
  </h1>,
  <h2>
    Want to get in touch? We'd love to hear from you. Here's how you can contact us.
  </h2>,
  <article
    className="contact-article"
  >
    <section>
      <img
        alt=""
        src="assets"
      />
      <div
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
        Talk to us
      </div>
      <div>
        Do you have any questions, or you want to call delivery by phone?
        <br />
        Just pick up the phone to call us for more information.
      </div>
      <div>
        Hotline: 
        <a
          href="tel:19002707"
        >
          1900-2707
        </a>
      </div>
    </section>
    <section>
      <img
        alt=""
        src="assets"
      />
      <div
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
        Chat with us
      </div>
      <div>
        Do you have any questions, or you want to notify about new promotion?
        <br />
        Just like our Facebook fanpage or chat with us on Facebook.
      </div>
      <div>
        Join our fanpage
         
        <a
          href="https://www.facebook.com/pycozza"
          rel="noopener noreferrer"
          target="_blank"
        >
          here
        </a>
        .
      </div>
    </section>
  </article>,
]
`;
